ONSdigital/dp-frontend-search-controller
# Remove old search bar and use default one

### What

Remove the bespoke (old) search bar that was taken from sixteens and use the default search bar template from the dp-renderer.

updated the controller to now point at the dp-design-tool

### How to review

- pull and run the latest dp-design-system (npm run dev)
- run the new search controller and see if the search bar is consistent with the one on the homepage

### Who can review

Anyone
